server/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -192,7 +193,8 @@
 
     protected volatile ShardRouting shardRouting;
     protected volatile IndexShardState state;
-    protected volatile long primaryTerm;
+    protected volatile long pendingPrimaryTerm;
```

Comment:
can we have a comment pointing people to the java docs of `getPendingPrimaryTerm` for explanation of what it means?

Reply:
fixed in 4b82ca7f770